concent_api/conductor/tests/test_service.py

```diff
@@ -1,12 +1,17 @@
 import mock
+import pytest
+from assertpy import assert_that
+
+from golem_messages.factories.tasks import ComputeTaskDefFactory
 
 from common.helpers import get_current_utc_timestamp
 from common.helpers import parse_timestamp_to_utc_datetime
 from conductor.exceptions import VerificationRequestAlreadyInitiatedError
-from conductor.models import ResultTransferRequest
+from conductor.models import ResultTransferRequest, BlenderCropScriptParameters
```

Comment:
You should move it to new line but this is only style issue.

Reply:
Done